test/com/core/tictactoe/BoardTests.java

```diff
@@ -0,0 +1,171 @@
+package com.core.tictactoe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+
```

Comment:
Just one line break between imports and start of Class definition.

Reply:
Done.